contracts/metatx/ERC2771Forwarder.sol

```diff
@@ -97,8 +107,8 @@ contract ERC2771Forwarder is EIP712, Nonces {
      * receiver is provided.
      */
     function verify(ForwardRequestData calldata request) public view virtual returns (bool) {
-        (bool alive, bool signerMatch, ) = _validate(request);
-        return alive && signerMatch;
+        (bool unexpired, bool signerMatch, ) = _validate(request);
```

Comment:
Personally I would go with "active" instead of "unexpired".

Reply:
Fixed in [7f6da22](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)